.coderabbit.yaml

```
@@ -0,0 +1,5 @@
+reviews:
+  tools:
+    # oxlint does not run if biome is enabled
+    ruff: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)